Src/FluentAssertions/Common/CSharpAccessModifierExtensions.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace FluentAssertions.Common
 {
-    public static class CSharpAccessModifierExtensions
+    internal static class CSharpAccessModifierExtensions
```

Comment:
How a test like this pseudo code?
```cs
AllTypes
    .UnderNamespace("FluentAssertions.Common")
    .ThatAreClasses
    .Should().BeInternal();
```

Reply:
I don't get your point. Those extensions where only used by internal code.

Reply:
My idea was a test to verify that types in the `Common` namespace indeed are marked as `internal`.
But it might be superfluous given we have the API approval tests to verify API changes.